modules/computation/variables.tf

```diff
@@ -65,3 +65,21 @@ variable "subnet2_id" {
   type        = string
   description = "The second private subnet used for redundancy"
 }
+
+variable "launch_template_http_endpoint" {
+  type        = string
+  description = "Whether the metadata service is available. Can be 'enabled' or 'disabled'"
+  default     = "disabled"
+}
+
+variable "launch_template_http_tokens" {
+  type        = string
+  description = "Whether or not the metadata service requires session tokens, also referred to as Instance Metadata Service Version 2 (IMDSv2). Can be 'optional' or 'required'"
+  default     = "optional"
+}
+
+variable "launch_template_http_put_response_hop_limit" {
+  type        = number
+  description = "The desired HTTP PUT response hop limit for instance metadata requests. Can be an integer from 1 to 64"
+  default     = 1
```

Comment:
Is 1 a good default? I vaguely remember that it needs to be 2 for docker since docker networking adds another hop

Reply:
I don't know tbh. I just know TF defaults to 1, so it seems to make sense to do the same here to avoid unexpected changes in people's plans. Can bump to 2 if that's preferred.

Reply:
I don't think would matter for the current users of the module since its only IMDSv1 and IMDSv1 doesn't use the hop limit. Can you test it with IMDSv2 tokens required and hop limit set to 1, does it work?

Reply:
You're right, I increased the default to 2.